cartridges/app_custom_product_ratings/cartridge/controllers/Product.js

```diff
@@ -0,0 +1,120 @@
+"use strict";
+
+var server = require("server");
+var page = module.superModule;
+server.extend(page);
+var Site = require("dw/system/Site");
+
+// server.append("Show", function (req, res, next) {
+//   var randomStr = Site.getCurrent().getCustomPreferenceValue("randomStr");
+
+//   var viewData = res.getViewData();
+//   viewData.randomStr = randomStr;
+
+//   res.setViewData(viewData);
+//   next();
+// });
+
+var getCookie = function (name) {
```

Comment:
you have this function in your previous controller, extract it in a helper and include it in both controllers